app/models/manageiq/providers/ansible_playbook_workflow.rb

```diff
@@ -9,4 +9,17 @@ def launch_runner
 
     Ansible::Runner.run_async(env_vars, extra_vars, playbook_path, kwargs)
   end
+
+  private
+
+  def verify_options
+    if !!options[:configuration_script_source_id] ^ !!options[:playbook_relative_path]
```

Comment:
Should we add an additional condition for `playbook_path`?

Really the happy path is this, right? `options[:playbook_path] || (options[:configuration_script_source_id] && options[:playbook_relative_path])`

Reply:
I guess I don't want to fail if we *could* succeed. For example the case where someone passes `:playbook_path` and also `:playbook_relative_path` shouldn't fail, but maybe should warn.

Reply:
Yeah, I could do that...I skipped it because on the roles side it's trickier, but I could probably figure something out.